volume/volume.go

```diff
@@ -73,6 +73,12 @@ type LabeledVolume interface {
 	Volume
 }
 
+// OptionedVolume wraps a Volume with driver defined options
+type OptionedVolume interface {
```

Comment:
Maybe we can replace all these 1-off interfaces with a single `VolumeWithDetails`... maybe a better name.
But all volumes will have it since we are wrapping all the volumes.